ispcrt/cmake/Finddpcpp_compiler.cmake

```diff
@@ -25,6 +25,8 @@ if (DPCPP_COMPILER)
 
     # oneAPI distributions ship the llvm utility binaries under bin-llvm
     set(DPCPP_LLVM_BIN_HINT "bin-llvm")
+    # or under bin/compiler (since oneAPI release 2023.3?)
+    set(DPCPP_LLVM_BIN_HINT2 "bin/compiler")
```

Comment:
I suggest to combine both hints together: `set(DPCPP_LLVM_BIN_HINT "bin-llvm" "bin/compiler")`

Reply:
fixed